drv/cosmo-hf/src/hf.rs

```diff
@@ -554,11 +729,13 @@ impl idl::InOrderHostFlashImpl for ServerImpl {
 
 impl NotificationHandler for ServerImpl {
     fn current_notification_mask(&self) -> u32 {
-        0
+        notifications::TIMER_MASK
```

Comment:
Should we only request this notification if the hash is in progress?  I guess it's harmless to always have the bit active...

Reply:
This seems to match what we do throughout hubris